les/handler.go

```diff
@@ -242,7 +243,7 @@ func (pm *ProtocolManager) Stop() {
 	// will exit when they try to register.
 	pm.peers.Close()
 
-	// Wait for any process action
+	// Wait for any process action. Wait should be executed after all pm.wg.Add()
```

Comment:
Please remove this comment.

Reply:
Thanks. This and upper changes has been done.